packages/check/match_test.ts

```diff
@@ -0,0 +1,99 @@
+import { Tinytest } from "meteor/tinytest";
+import { check, Match } from "meteor/check";
+
+Tinytest.add("Check - TypeScript types - basic check function", (test) => {
```

Comment:
would be bteer to have a test for "general types" so test string, number, obj...